frostsnap_coordinator/src/check_share.rs

```diff
@@ -67,7 +67,7 @@ impl CheckShareProtocol {
 
 impl UiProtocol for CheckShareProtocol {
     fn cancel(&mut self) {
-        self.abort("loading share cancelled".to_string())
+        self.abort("Loading share cancelled".to_string())
```

Comment:
I capitalized this because we're currently showing this error on the snackbar, probably will move to better error messaging here, but a quick style-fix for now.